### PR TITLE
PAS440-enhanced-logging

### PR DESCRIPTION
**Pull request by jrmccannon**
### Ticket

- Closes [PAS-440](https://bitwarden.atlassian.net/browse/PAS-440)

### Description
This adds some more information to the datadog logs and writes out some request info in the event of an improper POST.

### Shape
Added Ip, User, and UserAgent to all datadog logs via serilog.
Added Form to the log information on an improper submission be either hidden field or dupliate user.

### Checklist
I did the following to ensure that my changes were tested thoroughly:
- Was able to log in and use AdminConsole normally
- Form was correctly logged in the event of the duplicate submission
- Form was correctly logged in the event of an invalid submission

I did the following to ensure that my changes do not introduce security vulnerabilities:

N/A

[PAS-440]: https://bitwarden.atlassian.net/browse/PAS-440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/AdminConsole/Pages/Organization/Create.cshtml.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public class Create : PageModel
     private readonly MagicLinkSignInManager<ConsoleAdmin> _magicLinkSignInManager;
     private readonly IEventLogger _eventLogger;
     private readonly ILogger<Create> _logger;
+    private readonly IHttpContextAccessor _httpContextAccessor;
```
Comment (Tyrrrz) [inline src/AdminConsole/Pages/Organization/Create.cshtml.cs:22]: Where is this referenced in code? I can't see it being used 🤔

Comment (jrmccannon) [reply src/AdminConsole/Pages/Organization/Create.cshtml.cs:22]: Its not :D

**Review by abergs — COMMENTED** [review]
lgtm!

**Review by abergs — APPROVED** [review]
lgtm!

**Pull request merged**